custom_components/tahoma/entity.py

```diff
@@ -144,6 +145,14 @@ class OverkizNumberDescription(NumberEntityDescription):
     state: Callable[[str], bool] = lambda state: state
 
 
+@dataclass
+class OverkizSelectDescription(SelectEntityDescription):
+    """Class to describe an Overkiz select entity."""
+
+    options: list[str] = None
+    select_option: Callable[[str], bool] = lambda option, execute_command: None
```

Comment:
I’m not sure about the execute_command parameter. In which case do you think it can be something else than the current execute_command?

We can add it later, if needed.

Reply:
What do you mean? I need the execute_command parameter to call the function. See the code for the gate, I have to execute 3 different commands depending on the selected option (instead of just one command).